etc/initsystem/icinga2.service.cmake

```diff
@@ -5,6 +5,7 @@ After=syslog.target network-online.target postgresql.service mariadb.service car
 
 [Service]
 Type=notify
+NotifyAccess=all
```

Comment:
Given that this will also allow check plugins to send status updates, maybe it would make sense to at least strip the `NOTIFY_SOCKET` variable from their environment (do we already have a mechanism for doing something like that?) to prevent doing so by accident?